clang/include/clang/Driver/Options.td

```diff
@@ -2680,6 +2680,11 @@ def fsycl_footer_path_EQ : Joined<["-"], "fsycl-footer-path=">,
 def fno_sycl_link_spirv : Flag<["-"], "fno-sycl-link-spirv">,
   Flags<[CoreOption]>, HelpText<"Disable adding of the default (spir64) triple "
   "when discovered in user specified objects and archives.">;
+def fsycl_max_parallel_jobs_EQ : Joined<["-"], "fsycl-max-parallel-link-jobs=">,
+  Flags<[CoreOption]>, Group<f_Group>,
+  HelpText<"Controls the parallelism of post device code link actions performed"
+  "by the toolchain, like generating SPIR-V files for each device image"
+  "or performing an AOT compilation for each device image">;
```

Comment:
Do we have to specify the experimental nature of the feature within this help text/in the UsersManual.md block of SYCL documentations?

Reply:
Yes, I think experimental tags need to be added for both help and documentation.

Reply:
Added the experimental marks and improved doc coverage within https://github.com/intel/llvm/pull/4543/commits/e5bb1352de63cf366a6360918e19a69a628f38f1.